pyproject.toml

```diff
@@ -14,39 +14,39 @@ include = ["v03_pipeline*"]
 exclude = ["v03_pipeline.bin", "v03_pipeline*test*"]
 namespaces = false
 
-[tool.black]
-line-length = 88
-skip-string-normalization = true
-extend-exclude = '''
-(
-    download_and_create_reference_datasets/v02
-    | gcloud_dataproc
-    | hail_scripts/computed_fields
-    | hail_scripts/elasticsearch
-    | hail_scripts/update_models
-    | hail_scripts/utils
-    | kubernetes
-    | luigi_pipeline/lib
-    | luigi_pipeline/seqr.*\.py$
-    | luigi_pipeline/tests/data
-)
-'''
+#[tool.black]
+#line-length = 88
+#skip-string-normalization = true
+#extend-exclude = '''
+#(
+#    download_and_create_reference_datasets/v02
+#    | gcloud_dataproc
+#    | hail_scripts/computed_fields
+#    | hail_scripts/elasticsearch
+#    | hail_scripts/update_models
+#    | hail_scripts/utils
+#    | kubernetes
+#    | luigi_pipeline/lib
+#    | luigi_pipeline/seqr.*\.py$
+#    | luigi_pipeline/tests/data
+#)
+#'''
 
-[tool.isort]
-include_trailing_comma = 'True'
-known_external_package = ['hail_scripts']
-line_length = 88
-profile = 'black'
-sections = ['FUTURE', 'STDLIB', 'THIRDPARTY', 'EXTERNAL_PACKAGE', 'FIRSTPARTY', 'LOCALFOLDER']
-skip_glob = [
-    'download_and_create_reference_datasets/v02/*',
-    'gcloud_dataproc/*',
-    'hail_scripts/computed_fields/*',
-    'hail_scripts/elasticsearch/*',
-    'hail_scripts/update_models/*',
-    'hail_scripts/utils/*',
-    'kubernetes/*'
-]
+#[tool.isort]
+#include_trailing_comma = 'True'
+#known_external_package = ['hail_scripts']
```

Comment:
I love that I played this weird game with the `hail_scripts` import and now it's just 💀